### PR TITLE
Tools/ci enhancements

### PR DESCRIPTION
**Pull request by goransle**

### CODE REVIEW

File: .github/workflows/highcharts-headless.yml [hunk 1]
```diff
@@ -17,12 +17,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: lts/*
+          node-version: 18
```
Comment (bre1470) [inline .github/workflows/highcharts-headless.yml:20]: Why not the latest LTS (v20)?

Comment (bre1470) [reply .github/workflows/highcharts-headless.yml:20]: I see now the `ts-node-unit-tests` issue. 👍

File: .circleci/config.yml [hunk 2]
```diff
@@ -11,3 +11,13 @@ jobs:
...
+  security_scan_cron:
+    triggers:
+      - schedule:
+          cron: "0 0 31 2 0" # February 31st
+          filters:
+            branches:
+              only:
+                - chore/scan-security
+    jobs:
+      - security_scan
```
Comment (bre1470) [inline .circleci/config.yml:23]: The setup for security scans needs to be created via repository settings.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

File: .circleci/config.yml (hunk 2 above)
Comment (goransle) [reply .circleci/config.yml:23]: To be clear: this bit is the workaround for the CircleCI "no workflows" error. (The scheduled date + the branch filter should ensure it the workflow never runs)

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**